.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,107 @@
+---
+name: 🐛 Bug Report
+description: Create a report to help us improve
+labels: ["bug", "needs-triage"]
+
+body:
+- type: markdown
+  attributes:
+    value: >
+      **Thank you for wanting to report a bug in pyhf!**
+
+
+      ⚠
+      Verify first that your issue is not [already reported on
+      GitHub][issue search].
+
+
+      [issue search]: https://github.com/scikit-hep/pyhf/issues?q=is%3Aopen+is%3Aissue+label%3Abug
+
+- type: textarea
+  attributes:
+    label: Summary
+    description: Explain the problem briefly below.
+    placeholder: >-
+      When I try to do X with pyhf and the following workspace, Y breaks or
+      Z happens in an unexpected manner.
+      Here are all the details I know about this problem.
+  validations:
+    required: true
+
+- type: textarea
+  attributes:
+    label: pyhf Version
+    description: >-
+      Paste verbatim output from `pyhf --version` below, under
+      the prompt line. Please don't wrap it with tripple backticks &mdash; your
+      whole input will be turned into a code snippet automatically.
+    render: console
+    placeholder: |
+      $ pyhf --version
+      pyhf, version 0.6.2
+  validations:
+    required: true
+
+- type: textarea
+  attributes:
+    label: OS / Environment
+    description: >-
+      Provide the name of your operating system and the output of `uname --all`.
+    placeholder: |
```

Comment:
Maybe render as console? Also, I'd add `cat /etc/os-release` and `cat /etc/lsb-release` (IIRC)

Reply:
During this portion, I wonder if we should provide a `pyhf debug` or similar CLI so users can copy/paste that output entirely.

Reply:
@kratsg want to Issue that for follow up?

Reply:
#1580